Makefile

```diff
@@ -36,25 +36,26 @@ lint:
 pre-commit: fmt lint full-test
 
 deps:
+	@mkdir -p target; 
```

Comment:
What is this line for?

Reply:
Directory target should be exist before download wasm's. For example: `make test` immideatly after clone repo